.github/workflows/pages.yaml

```diff
@@ -14,19 +14,26 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Configure pages
+        uses: actions/configure-pages@v5
+
+      - name: Setup python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.x"
+          cache: "pip"
```

Comment:
Just adding this line enables caching dependencies installed from `pyproject.toml`, which speeds up builds by like ~30s.